codice/ddf-ui
# Add option to auto-pan to shapes on search and map load

- Pans to shapes on search (if option is on, else nothing)
- Pans to shapes on map load (if option is on and shapes present, else pans home)

Testing:

1. Navigate to Search
2. Verify that auto-pan is turned on
  
    <img src=https://user-images.githubusercontent.com/22667297/106963782-58a70800-66fe-11eb-988c-c7259f3f961a.png width=60% height=60% />

3. Draw shapes on the map and verify that clicking the search button pans to the shapes (try this with each shape and with different buffer sizes)
4. Open 2D and 3D maps to verify that the map auto-pans to your shapes.
5. Remove all of your shapes, set a home location, and open 2D and 3D maps to verify that panning defaults to home when there are no shapes.
6. Turn off auto-pan and verify that it's no longer functioning
